keyboards/handwired/aranck/keymaps/turkishish/readme.md

```diff
@@ -1 +1,7 @@
 # The Turkishish keymap for Aranck
+
+This layout is optimized for my use cases with Turkish OS layout.
+
+You can flash the "turkishish" layout like:
+
+	make handwired/aranck:turkishish
```

Comment:
Using the `:flash` target will actually flash the compiled firmware onto the board, as long as the bootloader is set correctly in `rules.mk`.

```suggestion
	make handwired/aranck:turkishish:flash
```